src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -66,7 +66,7 @@ public function collect(Request $request, Response $response, ?\Throwable $excep
         $sessionMetadata = [];
         $sessionAttributes = [];
         $flashes = [];
-        if (!$request->attributes->getBoolean('_stateless') && $request->hasSession()) {
```

Comment:
the changes in the profiler must not be reverted (the profiler must not attempt to read the session for stateless requests, as it would make all requests stateful in dev, defeating the feature)

Reply:
Ok, I updated